integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/BerikInntektsmeldingServiceIT.kt

```diff
@@ -331,8 +331,8 @@ class BerikInntektsmeldingServiceIT : EndToEndTest() {
 
         val forespoersel =
             Forespoersel(
-                orgnr = orgnr.verdi,
-                fnr = bjarneBetjent.ident!!,
+                orgnr = orgnr,
+                fnr = bjarneBetjent.ident!!.let(::Fnr),
```

Comment:
Hvordan har det seg egentlig at `FullPerson` sitt felt `ident` er nullable (som gjør at vi må double-bange denne)? Gir det mening, eller er det bare noe gammel moro?

Reply:
Jeg vet ikke hvorfor det ble gjort, men det hadde vært bedre om den ikke var det. Vi bruker jo typisk fnr for å hente personinfo, så man skulle trodd at man alltid hadde fnr.